spec/core/plugin_spec.rb

```diff
@@ -29,6 +29,22 @@ class LogStash::Filters::LadyGaga < LogStash::Filters::Base
     expect(LogStash::Plugin.lookup("filter", "lady_gaga")).to eq(LogStash::Filters::LadyGaga)
   end
 
+  describe "#inspect" do
+    class LogStash::Filters::MyTestFilter < LogStash::Filters::Base
```

Comment:
Can we make this part of a helpers file ? or locate it somewhere else. Having the definition of a class here looks strange to me.